resetpassword/lib/__init__.py

```diff
@@ -58,16 +59,22 @@ def _plain_send_mail(sender, recipient, subject, body):
     sender_name, sender_addr = parseaddr(sender)
     recipient_name, recipient_addr = parseaddr(recipient)
 
-    sender_name = str(Header(unicode(sender_name), header_charset))
-    recipient_name = str(Header(unicode(recipient_name), header_charset))
+    sender_name = str(Header(text(sender_name), header_charset))
+    recipient_name = str(Header(text(recipient_name), header_charset))
 
     sender_addr = sender_addr.encode('ascii')
     recipient_addr = recipient_addr.encode('ascii')
 
+    if isinstance(sender_addr, bytes):
+        sender_addr = sender_addr.decode('utf-8')
```

Comment:
Not clear to me the reason for this check.
It looks like `sender_addr` will always be `bytes` as it's just previously decoded so it will always result in bytes.